tools/pipelines/templates/build-npm-package.yml

```
@@ -570,6 +579,15 @@ extends:
                       sbomEnabled: false
                       publishLocation: pipeline
 
+                - ${{ if eq(parameters.testCoverage, true) }}:
+                  - output: pipelineArtifact
+                    displayName: Publish Artifacts - code-coverage
+                    condition: and( succeeded(), ne(variables['Build.Reason'], 'PullRequest'), eq(${{ parameters.taskPublishCodeCoverageArtifacts }}, true) )
```

I think we can keep things simple here; if `parameters.testCoverage` is `true` for whatever reason, we can publish the analysis as an artifact. No one might use it (particularly, when this runs in the internal ADO project, no one will; at least today) but that's fine. I think this also means we can remove the new parameter entirely. The current condition that skips include-set-package-version can also remain as it is today. Thoughts?
```suggestion
                    condition: and( succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
```

---

Yes, modieifed it.